libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/navigation/DownloadUpdateListenerTest.kt

```diff
@@ -0,0 +1,55 @@
+package com.mapbox.services.android.navigation.v5.navigation
+
+import java.io.File
+import org.junit.Test
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.verify
+
+class DownloadUpdateListenerTest {
+
+    @Test
+    fun onFinishedDownloading_tarIsUnpacked() {
+        val tileUnpacker = mock<TileUnpacker>(TileUnpacker::class.java)
```

Comment:
We could remove explicit type arguments. The same thing happens in other variables below 👇

Reply:
agree, done